PWGHF/HFL/Tasks/taskSingleMuon.cxx

```diff
@@ -205,25 +215,38 @@ struct HfTaskSingleMuonSelection {
       VarManager::FillTrack<TMuonFillMap>(track, values);
 
       // compute DCAXY
-      // With DQ's muon DCA calculation applied, we no longer need calculate it ourselve.
       // Same for MC
       const auto dcaXY(std::sqrt(values[VarManager::kMuonDCAx] * values[VarManager::kMuonDCAx] + values[VarManager::kMuonDCAy] * values[VarManager::kMuonDCAy]));
 
-      // Before Muon Cuts
-      registry.fill(HIST("hMuBcuts"),
-                    values[VarManager::kPt],
-                    values[VarManager::kEta], dcaXY,
-                    values[VarManager::kCharge], track.p(),
-                    values[VarManager::kVtxZ],
-                    values[VarManager::kMuonTrackType], values[VarManager::kMuonChi2MatchMCHMFT], 0);
-      // After Muon Cuts
-      if (trackCut->IsSelected(values))
-        registry.fill(HIST("hMuAcuts"),
+      if (histMuonLite) {
+        // Before Muon Cuts
+        registry.fill(HIST("hMuLiteBcuts"),
+                      values[VarManager::kPt],
+                      values[VarManager::kEta], dcaXY,
+                      values[VarManager::kMuonTrackType], values[VarManager::kMuonChi2MatchMCHMFT]);
+        // After Muon Cuts
+        if (trackCut->IsSelected(values))
+          registry.fill(HIST("hMuLiteAcuts"),
+                        values[VarManager::kPt],
+                        values[VarManager::kEta], dcaXY,
+                        values[VarManager::kMuonTrackType], values[VarManager::kMuonChi2MatchMCHMFT]);
```

Comment:
You fill the lite histograms (like `hMuLiteBcuts`) only in `runMuonSel` and not in `runMuonSelMC`, whereas you fill the standard histograms (like `hMuBcuts`) in both functions. Is that intended?

Reply:
Yes, we do not need the MC lite information at present.